views/partials/footer.ejs

```diff
@@ -10,4 +12,5 @@
     <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
 </body>
-</html>
\ No newline at end of file
+</footer>
```

Comment:
Please follow the order. You see the `footer` is a child of the `body` tag. Please follow the order. I thought you'd see it when you correct the `html` tag. The site may crash at some point if the order of parent child component is not followed.

Reply:
@thebishaldeb I guess you are seeing my previous commits, I have modified it in my latest commit.